airbyte-integrations/connectors/source-recurly/source_recurly/spec.json

```diff
@@ -7,8 +7,15 @@
     "required": ["api_key"],
     "additionalProperties": false,
     "properties": {
+      "begin_time": {
```

Comment:
I don't think `begin_time` is the best name for this parameter

Reply:
I just reused the param name Recurly uses for consistency https://developers.recurly.com/api/v2021-02-25/#section/Getting-Started/Pagination#query-parameters. Does that make sense?